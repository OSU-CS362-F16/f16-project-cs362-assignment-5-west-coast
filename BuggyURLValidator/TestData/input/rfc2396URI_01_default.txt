http://www.ietf.org/rfc/rfc2396.txt
http://www.w3.org/Addressing/
ftp://ds.iternic.net/rfc/
ftp://ftp.is.co.za/rfc/rfc1808.txt
http://www.math.uio.no/faq/compression-faq/part1.html
https://foo.bar.org/