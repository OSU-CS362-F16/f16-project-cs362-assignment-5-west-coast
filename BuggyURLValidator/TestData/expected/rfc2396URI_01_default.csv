testcase,scheme,ALLOW_LOCAL_URLS,NO_FRAGMENTS,ALLOW_2_SLASHES,ALLOW_ALL_SCHEMES,regex,http://www.ietf.org/rfc/rfc2396.txt,http://www.w3.org/Addressing/,ftp://ds.iternic.net/rfc/,ftp://ftp.is.co.za/rfc/rfc1808.txt,http://www.math.uio.no/faq/compression-faq/part1.html,https://foo.bar.org/
0,{[http][https]},OFF,OFF,OFF,OFF,DEFAULT,true,true,false,false,true,true
1,{[http][https][ftp]},OFF,OFF,OFF,OFF,DEFAULT,true,true,true,true,true,true
2,{[http][https][ftp]},OFF,OFF,OFF,OFF,DEFAULT,true,true,true,true,true,true
3,{[http][https][ftp]},OFF,OFF,OFF,ON,DEFAULT,true,true,true,true,true,true
4,{[http][https][ftp]},OFF,OFF,ON,OFF,DEFAULT,true,true,true,true,true,true
5,{[http][https][ftp]},OFF,OFF,ON,ON,DEFAULT,true,true,true,true,true,true
6,{[http][https][ftp]},OFF,ON,OFF,OFF,DEFAULT,true,true,true,true,true,true
7,{[http][https][ftp]},OFF,ON,OFF,ON,DEFAULT,true,true,true,true,true,true
8,{[http][https][ftp]},OFF,ON,ON,OFF,DEFAULT,true,true,true,true,true,true
9,{[http][https][ftp]},OFF,ON,ON,ON,DEFAULT,true,true,true,true,true,true
10,{[http][https][ftp]},ON,OFF,OFF,OFF,DEFAULT,true,true,true,true,true,true
11,{[http][https][ftp]},ON,OFF,OFF,ON,DEFAULT,true,true,true,true,true,true
12,{[http][https][ftp]},ON,OFF,ON,OFF,DEFAULT,true,true,true,true,true,true
13,{[http][https][ftp]},ON,OFF,ON,ON,DEFAULT,true,true,true,true,true,true
14,{[http][https][ftp]},ON,ON,OFF,OFF,DEFAULT,true,true,true,true,true,true
15,{[http][https][ftp]},ON,ON,OFF,ON,DEFAULT,true,true,true,true,true,true
16,{[http][https][ftp]},ON,ON,ON,OFF,DEFAULT,true,true,true,true,true,true
17,{[http][https][ftp]},ON,ON,ON,ON,DEFAULT,true,true,true,true,true,true
18,{[http][https]},OFF,OFF,OFF,OFF,DEFAULT,true,true,false,false,true,true
19,{[http][https]},OFF,OFF,OFF,ON,DEFAULT,true,true,true,true,true,true
20,{[http][https]},OFF,OFF,ON,OFF,DEFAULT,true,true,false,false,true,true
21,{[http][https]},OFF,OFF,ON,ON,DEFAULT,true,true,true,true,true,true
22,{[http][https]},OFF,ON,OFF,OFF,DEFAULT,true,true,false,false,true,true
23,{[http][https]},OFF,ON,OFF,ON,DEFAULT,true,true,true,true,true,true
24,{[http][https]},OFF,ON,ON,OFF,DEFAULT,true,true,false,false,true,true
25,{[http][https]},OFF,ON,ON,ON,DEFAULT,true,true,true,true,true,true
26,{[http][https]},ON,OFF,OFF,OFF,DEFAULT,true,true,false,false,true,true
27,{[http][https]},ON,OFF,OFF,ON,DEFAULT,true,true,true,true,true,true
28,{[http][https]},ON,OFF,ON,OFF,DEFAULT,true,true,false,false,true,true
29,{[http][https]},ON,OFF,ON,ON,DEFAULT,true,true,true,true,true,true
30,{[http][https]},ON,ON,OFF,OFF,DEFAULT,true,true,false,false,true,true
31,{[http][https]},ON,ON,OFF,ON,DEFAULT,true,true,true,true,true,true
32,{[http][https]},ON,ON,ON,OFF,DEFAULT,true,true,false,false,true,true
33,{[http][https]},ON,ON,ON,ON,DEFAULT,true,true,true,true,true,true
34,{[http][https][ftp]},OFF,OFF,OFF,OFF,^[\w-\\.]*$,true,true,true,true,true,true
35,{[http][https][ftp]},OFF,OFF,OFF,ON,^[\w-\\.]*$,true,true,true,true,true,true
36,{[http][https][ftp]},OFF,OFF,ON,OFF,^[\w-\\.]*$,true,true,true,true,true,true
37,{[http][https][ftp]},OFF,OFF,ON,ON,^[\w-\\.]*$,true,true,true,true,true,true
38,{[http][https][ftp]},OFF,ON,OFF,OFF,^[\w-\\.]*$,true,true,true,true,true,true
39,{[http][https][ftp]},OFF,ON,OFF,ON,^[\w-\\.]*$,true,true,true,true,true,true
40,{[http][https][ftp]},OFF,ON,ON,OFF,^[\w-\\.]*$,true,true,true,true,true,true
41,{[http][https][ftp]},OFF,ON,ON,ON,^[\w-\\.]*$,true,true,true,true,true,true
42,{[http][https][ftp]},ON,OFF,OFF,OFF,^[\w-\\.]*$,true,true,true,true,true,true
43,{[http][https][ftp]},ON,OFF,OFF,ON,^[\w-\\.]*$,true,true,true,true,true,true
44,{[http][https][ftp]},ON,OFF,ON,OFF,^[\w-\\.]*$,true,true,true,true,true,true
45,{[http][https][ftp]},ON,OFF,ON,ON,^[\w-\\.]*$,true,true,true,true,true,true
46,{[http][https][ftp]},ON,ON,OFF,OFF,^[\w-\\.]*$,true,true,true,true,true,true
47,{[http][https][ftp]},ON,ON,OFF,ON,^[\w-\\.]*$,true,true,true,true,true,true
48,{[http][https][ftp]},ON,ON,ON,OFF,^[\w-\\.]*$,true,true,true,true,true,true
49,{[http][https][ftp]},ON,ON,ON,ON,^[\w-\\.]*$,true,true,true,true,true,true
50,{[http][https]},OFF,OFF,OFF,OFF,^[\w-\\.]*$,true,true,false,false,true,true
51,{[http][https]},OFF,OFF,OFF,ON,^[\w-\\.]*$,true,true,true,true,true,true
52,{[http][https]},OFF,OFF,ON,OFF,^[\w-\\.]*$,true,true,false,false,true,true
53,{[http][https]},OFF,OFF,ON,ON,^[\w-\\.]*$,true,true,true,true,true,true
54,{[http][https]},OFF,ON,OFF,OFF,^[\w-\\.]*$,true,true,false,false,true,true
55,{[http][https]},OFF,ON,OFF,ON,^[\w-\\.]*$,true,true,true,true,true,true
56,{[http][https]},OFF,ON,ON,OFF,^[\w-\\.]*$,true,true,false,false,true,true
57,{[http][https]},OFF,ON,ON,ON,^[\w-\\.]*$,true,true,true,true,true,true
58,{[http][https]},ON,OFF,OFF,OFF,^[\w-\\.]*$,true,true,false,false,true,true
59,{[http][https]},ON,OFF,OFF,ON,^[\w-\\.]*$,true,true,true,true,true,true
60,{[http][https]},ON,OFF,ON,OFF,^[\w-\\.]*$,true,true,false,false,true,true
61,{[http][https]},ON,OFF,ON,ON,^[\w-\\.]*$,true,true,true,true,true,true
62,{[http][https]},ON,ON,OFF,OFF,^[\w-\\.]*$,true,true,false,false,true,true
63,{[http][https]},ON,ON,OFF,ON,^[\w-\\.]*$,true,true,true,true,true,true
64,{[http][https]},ON,ON,ON,OFF,^[\w-\\.]*$,true,true,false,false,true,true
65,{[http][https]},ON,ON,ON,ON,^[\w-\\.]*$,true,true,true,true,true,true
66,{[http][https][ftp]},OFF,OFF,OFF,OFF,DEFAULT,true,true,true,true,true,true
